assets/queries/k8s/saTokenSecretQuery/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "ServiceAccount_Access_Secrets",
+  "queryName": "ServiceAccount_Access_Secrets",
```

Comment:
The query name should explain the problem at hand, something like: "ServiceAccount Allow Access To Secrets